addons/escoria-core/game/core-scripts/save_data/esc_save_manager.gd

```diff
@@ -65,6 +68,12 @@ func save_game_exists(id: int) -> bool:
 # - id: integer suffix of the savegame file
 # - p_savename: name of the savegame
 func save_game(id: int, p_savename: String):
+	if not save_enabled:
+		escoria.logger.report_warnings(
+			"esc_save_data_resources.gd",
+			["Save requested while saving is not possible. Save canceled."])
```

Comment:
I wonder if we need a warning for that. I'd go with a debug message perhaps.

Reply:
Done